src/app/personside/infotabs/ytelser/sykepenger/SykepengerEkspanderbartPanel.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import EkspanderbartYtelserPanel from '../felles-styling/EkspanderbartYtelserPanel';
+import { Sykmeldingsperiode } from '../../../../../models/ytelse/sykepenger';
+import { formaterDato } from '../../../../../utils/dateUtils';
+
+interface Props {
+    sykepenger: Sykmeldingsperiode;
+}
+
+function SykepengerEkspanderbartpanel({sykepenger}: Props) {
+
+    const tittelTillegsInfo = [
+        'ID-dato: ' + formaterDato(sykepenger.sykmeldtFom)
+    ];
+
+    return (
+        <EkspanderbartYtelserPanel tittel="Sykepenger" tittelTillegsInfo={tittelTillegsInfo}>
+            {JSON.stringify(sykepenger)}
```

Comment:
Dette blir vakker layout, ikke endre det :P

Reply:
Mmm, less is more